syntax = "proto3";

package dev_observer.api.storage.local;

import "dev_observer/api/types/processing.proto";
import "dev_observer/api/types/repo.proto";
import "dev_observer/api/types/config.proto";
import "dev_observer/api/types/sites.proto";

message LocalStorageData {
  repeated dev_observer.api.types.repo.GitRepository git_repos = 1;
  repeated dev_observer.api.types.processing.ProcessingItem processing_items = 2;
  dev_observer.api.types.config.GlobalConfig global_config = 3;
  repeated dev_observer.api.types.sites.WebSite web_sites = 4;
  repeated dev_observer.api.types.processing.ProcessingItemResult processing_results = 5;
  repeated dev_observer.api.types.repo.RepoToken tokens = 6;
}
